js/components/ImportIndicatorsPopover.js

```diff
@@ -1,142 +1,271 @@
-import React from 'react';
+import React, { useState, useContext, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { BootstrapPopoverButton } from './helpPopover';
 import api from '../apiv2';
 
-class ImportIndicatorsPopover extends React.Component {
-    // These define the different cases/views of the Popover to switch between.
-    DOWNLOAD_UPLOAD = 0;
-    FEEDBACK = 1;
-    CONFIRM = 2;
-    SUCCESS = 3;
+export const ImportIndicatorsContext = React.createContext();
+
+export class ImportIndicatorsButton extends BootstrapPopoverButton {
+    popoverName = "importIndicators"
+    popoverTitle = "Import indicators"
 
     constructor(props) {
         super(props);
         this.state = {
-            status: this.DOWNLOAD_UPLOAD
+            levelStore: this.props.levelStore, 
```

Comment:
As we discussed, pass this directly into Context rather than running it through state.